### ISSUE TITLE
LXQt with Kwin in Debian 12- Is it Wayland?

### ISSUE THREAD

**Issue by subhaniminhas** [stefonarch/LXQt-Wayland-files#4]
Since Kwin is botha. Compositor as well as a Window Manager, so technically, if I start Lxqt with Kwin in Debain 12, am I starting a lxqrt Wayland session? 
In Debian 12, there are Kwin-X11 and simple Kwin session types available, which I presume to be a Wayland session. Although lxqt does not specify itself to be Wayland compatible while starting the session, unlike Gnome or KDE, which have specific session for Wayland. 
So the question is, do I need your project to start lxqt in Debain 12, or do I simply wait and let Debian 12 rollout with above mentioned settings. 
Thx
Happy Coding

**Comment by stefonarch** [comment]
> Since Kwin is botha. Compositor as well as a Window Manager, so technically, if I start Lxqt with Kwin in Debain 12, am I starting a lxqrt Wayland session?

No, `lxqt-session` cannot start any wayland compositor which has to be started first and will start the session, for kwin see https://github.com/stefonarch/LXQt-Wayland-files/blob/c3cb372c2a5dbb932736776eed691e709b2aa0ab/start_scripts/startlxqtkwin#L84

>So the question is, do I need your project to start lxqt in Debain 12, or do I simply wait and let Debian 12 rollout with above mentioned settings.

You have to use those files and settings from here, but IMO the kwin session is not  quite usable as there is no working taskmanager.  At the moment for an usable LXQt wayland session I recommend wayfire-git with the lxqt-session-plugin.

**Comment by stefonarch** [comment]
Outdated, LXQt 2.2 works perfectly with kwin_wayland
